source/includes/fundamentals/code-snippets/builders/AggBuilders.java

```diff
@@ -88,6 +89,15 @@ private void aggregationExample() {
         // end sampleAggregation
     }
 
+    private void setWindowFieldsStage() {
+        // begin setWindowFields
+        Window pastMonth = Windows.timeRange(-1, Windows.Bound.CURRENT, Unit.MONTH);
```

Comment:
I have realized that `Unit` was renamed to `MongoTimeUnit`. I also need to update it in the old Java driver docs.

Reply:
Thanks, I'll update this now.